gclient/src/api/mod.rs

```diff
@@ -28,11 +28,11 @@ use crate::{
 };
 use error::*;
 use gsdk::{ext::sp_runtime::AccountId32, signer::Signer, Api};
-use std::{marker::PhantomData, ops::Deref};
+use std::{ffi::OsStr, sync::Arc};
 
 /// The API instance contains methods to access the node.
 #[derive(Clone)]
-pub struct GearApi(Signer);
+pub struct GearApi(Signer, Option<Arc<Node>>);
```

Comment:
I'd change it to struct
```suggestion
pub struct GearApi {
    signer: Signer, 
    node: Option<Arc<Node>>
}
```

Reply:
@shamilsan are you sure about this. I have done it locally, but it imposes a lot of changes.

Reply:
As it doesn't affect external interface (only internal implementation), we can do it later, ok.